src/app/src/main/kotlin/org/imperial/mrc/hint/security/tokens/OneTimeTokenAuthenticator.kt

```diff
@@ -1,18 +1,23 @@
 package org.imperial.mrc.hint.security.tokens
 
 import com.nimbusds.jwt.JWT
+import org.imperial.mrc.hint.AppProperties
 import org.pac4j.core.credentials.TokenCredentials
 import org.pac4j.core.exception.CredentialsException
 import org.pac4j.jwt.config.signature.SignatureConfiguration
 import org.pac4j.jwt.credentials.authenticator.JwtAuthenticator
+import org.springframework.context.annotation.Configuration
 
+@Configuration
```

Comment:
I rearranged the constructors of this and related classes to make better use of Spring IoC for testability